pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -546,6 +546,7 @@ type Config struct {
 	PropagateProjectAnnotations    []string        `json:"propagateProjectAnnotations" name:"propagate-project-annotation" usage:"The list of keys of annotations to propagate from acorn project to app namespaces"`
 	PropagateProjectLabels         []string        `json:"propagateProjectLabels" name:"propagate-project-label" usage:"The list of keys of labels to propagate from acorn project to app namespaces"`
 	ManageVolumeClasses            *bool           `json:"manageVolumeClasses" name:"manage-volume-classes" usage:"Manually manage volume classes rather than sync with storage classes, setting to 'true' will delete Acorn-created volume classes"`
+	VolumeSizeDefault              string          `json:"volumeSizeDefault" name:"volume-size-default" usage:"Set the default size for acorn volumes. Accepts storage suffixes (K, M, G, etc) and \".\" and \"_\" seperators (default 0)" short:"v"`
```

Comment:
Imo we should not have `-v` as short for this. I'm so used to thinking that `-v` means verbose, and for a very specific configuration parameter like this, I don't think it really makes sense to have a short version.

Also, this is more of a nit, but I would also add to the list of accepted suffixes `Mi` and `Gi` to indicate that the base 2 suffixes are supported as well.

Reply:
The -v was not intentional. That was copilot 😅

Reply:
Updated!